QCamera/QCAppDelegate.swift

```diff
@@ -39,6 +39,7 @@ class QCAppDelegate: NSObject, NSApplicationDelegate, QCUsbWatcherDelegate {
     var devices: [AVCaptureDevice]!;
     var captureSession: AVCaptureSession!;
     var captureLayer: AVCaptureVideoPreviewLayer!;
+    var stillImageOutput: AVCaptureStillImageOutput?;
```

Comment:
Semicolons in Swift aren't needed, but don't give a error either. It's just "good" practice not to use them in swift :-)
```suggestion
    var stillImageOutput: AVCaptureStillImageOutput?
```